### PR TITLE
handle initialize with entry for Gtk::ComboBoxText

### PR DESCRIPTION
**Pull request by cedlemo**
Allow user to create a Gtk::ComboBoxText with entry using :entry => true

### CODE REVIEW

File: gtk3/test/test_gtk_combo_box_text.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      test "no others" do
+        combo_box_text = Gtk::ComboBoxText.new(:entry => true)
+        assert do
+          combo_box_text.has_entry?
+        end
+      end
+    end
+    sub_test_case("no :entry") do
+      test "no others" do
+        combo_box_text = Gtk::ComboBox.new
```
Comment (kou) [inline gtk3/test/test_gtk_combo_box_text.rb]: Typo: `Text` is missing in `Gtk::ComboBox`.

**Comment by cedlemo** [comment]
Oups sorry !

I was wondering if I should remove the<code>sub_test_case</code> for  :entry true or false and just create two <code> test "entry =>true"</code> and <code>test "entry => false"</code>.

**Comment by kou** [comment]
> I was wondering if I should remove the `sub_test_case` for `:entry true` or `false` and just create two `test "entry =>true"` and `test "entry => false"`.

It's better. Could you do so?

**Comment by cedlemo** [comment]
Done  .

**Comment by kou** [comment]
Merged!

**Pull request merged**